core/src/main/java/software/amazon/awssdk/runtime/StandardMemberCopier.java

```diff
@@ -74,6 +74,19 @@ public static ByteBuffer copy(ByteBuffer bb) {
         if (bb == null) {
             return null;
         }
-        return bb.duplicate();
+
+        ByteBuffer copy = bb.isDirect() ? ByteBuffer.allocateDirect(bb.capacity()) : ByteBuffer.allocate(bb.capacity());
+
+        // Copy content
+        ByteBuffer toCopy = bb.asReadOnlyBuffer();
+        toCopy.rewind();
```

Comment:
I didn't know there was a BinaryUtils. I'll take a look!

I did a rewind because without it, we're not actually copying the input buffer, and that could surprise someone that expects it to actually be a copy. For example:
```Java
ByteBuffer myBuffer = ...;
ByteBuffer copiedBuffer = Model.builder().byteBuffer(myBuffer).build().byteBuffer();

myBuffer.rewind();
copiedBuffer.rewind();

boolean equal = myBuffer.compareTo(copiedBuffer) == 0;
System.out.println(equal); // Prints either "false" or "true" depending on the state of myBuffer when it was copied.
```

In most cases they'll be giving us a buffer that is already reading from the beginning, so I didn't think the optimization of maybe not having to copy some data was worth the chance of surprising a customer.